test-packages/e2e-tests/test/http-agent.spec.ts

```diff
@@ -1,43 +1,42 @@
 import { createServer, Server } from 'https';
 import { promisify } from 'util';
-import {DestinationCertificate} from "@sap-cloud-sdk/connectivity/dist/scp-cf";
-import {Destination} from "@sap-cloud-sdk/connectivity";
-import {executeHttpRequest} from "@sap-cloud-sdk/http-client";
+import { DestinationCertificate } from '@sap-cloud-sdk/connectivity/dist/scp-cf';
```

Comment:
[q] It seems this file was not under the radar of lint, as it did not fail.
How did you fix the lint? Manual fix?
Should we consider adding specs to the lint scope?

Reply:
I fixed it manually here. 
The package.json in e2e-tests project doesn't have a lint command in scripts. I'll add it.

Reply:
I think added the certificate here for testing and did not set a very long lifetime. Sorry for that. Please use a self signed certificate with maximum lifetime here.